app/customer/routes.py

```diff
@@ -131,7 +132,8 @@ def search():
         session['search_tag'] = search_tag
 
     page = request.args.get('page', 1, type=int)
-    customers = db.paginate(db.select(Customer).where(Customer.name.like('%' + search_tag + '%')),
+    #customers = db.paginate(db.select(Customer).where(Customer.name.like('%' + search_tag + '%')),
+    customers = db.paginate(db.select(Customer, Email).where(or_(Customer.name.match(search_tag), Email.email.match(search_tag))), ## WORKS KIND OF
```

Comment:
@Wcarey7, this is the part that kind of works.  It is working as in it's utilizing FTS in MySQL, but of course it only works for the Customer's name.  The `or_()` didn't work like I thought it would.  Any ideas here (along with my comment [here](https://chat.google.com/room/AAAA0wDkRXE/9mp66WGrPSU/lQcNxg9_CAo?cls=10))?

Reply:
Thinking.